src/js/components/LottoNumbers.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @param {Element} $el
+ * @param {Object} props
+ * @param {[[Number]]} props.lottoNumbers
+ * @constructor
+ */
+export function LottoNumbers($el, props) {
+
+    /**
+     * @type {Object}
+     * @property {Boolean} showNumbersView
+     */
+    let state = {
+        showDetailNumbers: false,
+    };
+
+    const setState = (nextState) => {
+        state = {
+            ...state,
+            ...nextState,
+        };
+        render();
+    };
+
```

Comment:
setState에서 nextState 받아오실 때, 변경될 상태만 받아오고 계신데요!! 이러면 나중에 매개변수 타입 정의할 때 조금 까다로워지고, 매개변수가 매번 달라지는 점이 좀 불안정한 것 같아요 !! 기존 state에 없는 state가 들어와서 object에 추가될 수도 있으니까요!! 그래서 아예 아래와 같이 구현해보는건 어떠신가요..?!?
```suggestion
    const setState = (nextState) => {
       state = nextState;
        render();
    };
    
    // setState 쓸 경우
    setState({...state, howDetailNumbers: !state.showDetailNumbers})

```